tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2015",
-    "lib": ["dom", "es2015"],
+    "target": "es6",
```

Comment:
is this compatible with react-native's JS VM?

also I added the `dom` library to get the definition of `fetch`

Reply:
es6 is a synonim for es2015 so by the runtime point of view this does not change anything. 

but... when you use es2015 *and* specify the "lib" setting, for some reason the es2015 libraries are not included as a whole (that's why you need to add 'dom' in that case). In this specific case you should have included also the `es2015.collection` libray (to support `ReadonlySet`) and so on.

If you don't specify "lib" manually everything gets included as a whole by default:

> For --target es6 => es6, dom, dom.iterable, scripthost

Reply:
ref. https://stackoverflow.com/a/47204778/21424